cmd/tesseract/README.md

```diff
@@ -201,7 +201,21 @@ resources TesseraCT spends on servicing duplicate requests.
 
 The `garbage_collection_interval` flag controls Tessera's Garbage Collection.
 It controls how often Tessera scans for partial tiles and entry bundles, and
-deletes them if a matching full tile or entry bundle has been published.
+deletes them if a matching full tile or entry bundle has been published. Each
+storage implementation keeps track of the highest entry index that was garbage
+collected. Every `garbage_collection_interval`, Tessera pre-computes the potential
+paths of [100](https://github.com/search?q=repo%3Atransparency-dev%2Ftessera+maxBundlesPerRun&type=code)
+partial entry bundle and partial tile directories. It then attempts to delete them
+one after another. If a path resolves to an existing partial entry bundle or tile
+directory, it is deleted. Otherwise, nothing happens and it moves to the next one.
```

Comment:
This is a bit focused on POSIX, maybe a more general description could be something like:

_Tessera examines the log sequentially in chunks, one chunk per `garbage_collection_interval`, removing any obsolete partial resources found within the current chunk. Currently, each chunk is defined as up-to 100 entry bundles along with the vertical "slice" of the merkle tree which covers these bundles._